src/modules/rlm_eap/eap.c

```diff
@@ -909,7 +909,7 @@ eap_session_t *eap_session_continue(eap_packet_raw_t **eap_packet_p, rlm_eap_t *
 		 *
 		 *	We must pass a NULL pointer to associate the
 		 *	the EAP_SESSION data with, else we'll break
-		 *	tunelled EAP, where the inner EAP module is
+		 *	tunnelled EAP, where the inner EAP module is
```

Comment:
I'm just using the GitHub desktop client..... sometimes ID get expanded,
other times not.  I'll see what I can do...then I'll push these again.
also, note, I'll go for 'tunneled' (as its far more common in the source
code and spelt that way in the RFCs...  however, tunelled was wrong anyway!
;-)

On 30 August 2016 at 09:57, Arran Cudbard-Bell <notifications@github.com>
wrote:

> In src/modules/rlm_eap/eap.c
> <https://github.com/FreeRADIUS/freeradius-server/pull/1701#discussion_r76755249>
> :
>
> > @@ -909,7 +909,7 @@ eap_session_t *eap_session_continue(eap_packet_raw_t **eap_packet_p, rlm_eap_t *
> >  		 *
> >  		 *	We must pass a NULL pointer to associate the
> >  		 *	the EAP_SESSION data with, else we'll break
> > -		 *	tunelled EAP, where the inner EAP module is
> > +		 *	tunnelled EAP, where the inner EAP module is
>
> it really is tunneled ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeRADIUS/freeradius-server/pull/1701/files/9ce5ac9c8201a0e7af6d20ced2f3ec036a7e44be#r76755249>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACE-VQBxV0l_YBcoGT2pvrKHp2tfKIQyks5qk_BogaJpZM4JwScQ>
> .
>